README.md

```diff
@@ -43,6 +43,11 @@ Elyra currently includes:
 The [Elyra Getting Started Guide](https://elyra.readthedocs.io/en/latest/getting_started/overview.html)
 includes more details on these features.
 
+## Try Elyra
+
+ You can try out some Elyra features in the following sandbox environments without having to install anything. 
+  - [Binder](https://mybinder.org/v2/gh/elyra-ai/elyra/master?urlpath=lab/tree/binder-demo) ([Learn more...](https://mybinder.readthedocs.io/en/latest/))
+
```

Comment:
How about you mention "Binder" around sandbox environments and add the link to binder, and on where you have the link, just copy the binder link from the top of the page ?

Reply:
I have made a quick update, if you are ok with that I am good to merge

Reply:
I had contemplated using the badge but decided against it because down the road we'll likely have sandbox environments for which there is no badge or the badge could look fundamentally different, which would result in an inconsistent look and feel. However, for now this is a theoretical problem so I agree the change is fine.